### PR TITLE
PHP: Autorelease Library

### PR DESCRIPTION
**Pull request by svix-frank**

### CODE REVIEW

File: .github/workflows/php-release.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+jobs:
+  packagist:
+    name: Update Packagist
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Update Packagist
+        run: |
+          curl -XPOST "https://packagist.org/api/update-package?username=svix&apiToken=$PACKAGIST_API_KEY" \
+            -H 'content-type:application/json' \
```
Comment (tasn) [inline .github/workflows/php-release.yml:17]:
```suggestion
            -H 'content-type: application/json' \
```
Burns my eyes. :P

File: .github/workflows/php-release.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+jobs:
+  packagist:
+    name: Update Packagist
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Update Packagist
+        run: |
+          curl -XPOST "https://packagist.org/api/update-package?username=svix&apiToken=$PACKAGIST_API_KEY" \
```
Comment (tasn) [inline .github/workflows/php-release.yml:16]: Seriously? They expect you to pass a token as a query param?

Comment (tasn) [inline .github/workflows/php-release.yml:16]:
```suggestion
          curl -X POST "https://packagist.org/api/update-package?username=svix&apiToken=$PACKAGIST_API_KEY" \
```

Comment (svix-frank) [reply .github/workflows/php-release.yml]: yep

**Review by tasn — APPROVED** [review]

**Pull request merged**